api/src/view_function.rs

```diff
@@ -22,6 +30,15 @@ pub struct ViewFunctionApi {
     pub context: Arc<Context>,
 }
 
+#[derive(ApiRequest, Debug)]
+pub enum ViewFunctionRequest {
+    #[oai(content_type = "application/json")]
+    Json(Json<ViewRequest>),
+
+    #[oai(content_type = "application/x.aptos.view_function+bcs")]
```

Comment:
I know we do it elsewhere but I find it weird that we have different content types for what is the same format. Why don't we just do `application/x-bcs` everywhere, it is after all the exact same content type.

Reply:
Well, that's now too late because the other one is already used for transactions with this format.  For backwards compatibility reasons this must stick with this.

Additionally, the `content_type` on the input tells you what kind of input you expect, I originally was wondering why is this like this, but this now tells us exactly the form that is coming in.